tutorials/nservicebus-step-by-step/4-publishing-events/tutorial.md

```diff
@@ -73,17 +70,12 @@ All the other considerations for command messages apply to events as well. Prope
 
 With events, you should be even more careful about putting too much information in an event message. Sometimes this complexity can't be avoided for commands, as the command receiver needs the information to do its job. That's manageable for commands because the sender and receiver are highly coupled already. For events, it's a different story. Since a publisher of an event does not know (or care) how many subscribers it has, it may not be possible to modify all of them if a change is required to the event.
 
-
 ## Handling events
 
 Create a handler class by implementing `IHandleMessages<T>` where `T` is the type of the event message.
 
 snippet: EventHandler
 
-> [!NOTE]
```

Comment:
Since all transports we currently support (since msmq is not out of support) supports native pubsub I felt that this part was not relevant for a tutorial